redact/tools/redact_file.py

```diff
@@ -2,10 +2,15 @@
 from pathlib import Path
 from typing import Optional, Union
 
-from redact.data_models import JobArguments, JobLabels
-from redact.redact_instance import RedactInstance
-from redact.redact_job import OutputType, RedactJob, ServiceType
-from redact.redact_requests import RedactRequests
+from redact.v3 import (
```

Comment:
No this will change when v4 is introduced. I am still trying to figure out how best to do it with different versions. But for now since there is only v3 I just kept it like this.